compiler/luci/partition/src/PartitionPGroups.cpp

```diff
@@ -195,6 +255,22 @@ std::unique_ptr<luci::PGroups> produce_pgroups(const luci::Module *source,
     }
   }
 
+  // handle for virtual nodes like multiple outputs
+  // these nodes should follow group of the input
```

Comment:
I'm not sure about what "follow" means. "follow group of the input" means the nodes are appended to the group of input?

Reply:
this is intended to explain that thiese nodes should be grouped to the same group of the input (owner).
maybe use `these nodes should be assigned to same group of the input` ?